repository/SmalltalkCI-Core.package/SCIMetacelloLoadSpec.class/instance/ensureAuthor.st

```diff
@@ -0,0 +1,4 @@
+initialize-release
+ensureAuthor
+	SmalltalkCI getAuthor ifEmpty: [
+		self class setAuthor: self class defaultAuthor ]
```

Comment:
It looks like you make the assumption that #getAuthor will return an empty string and that is not the case on GemStone ... again I can't tell, but the old way in prepareForCI looks like the author was preserved and right now I can't tell if the old way of preserving is still enforced